modules/slider/Slider.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { withStyles } from "@diana-ui/base";
+import { ThemeStyleSheetFactory, WithStylesProps } from "@diana-ui/types";
+import { LabelRegular } from "@diana-ui/typography";
+
+export interface ISliderProps {
```

Comment:
you should extend StandardProps<"input"> and then pass them to the input. This way we allow some standard input props that we can later be able to use, such as aria-label, classname,( disable, min, max, value, step ), etc